adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -288,12 +408,23 @@ - (nullable UIView *)renderBannerAdInView:(nonnull UIView *)bannerView
   return nil;
 }
 
-- (void)completeBannerAdViewForPlacementID:(nullable NSString *)placementID {
-  if (_isBannerPresenting) {
-    NSLog(@"Vungle: Triggering an ad completion call for %@", placementID);
-    _isBannerPresenting = NO;
+- (void)completeBannerAdViewForPlacementID:(nonnull id<GADMAdapterVungleDelegate>)delegate; {
```

Comment:
Should not be a ; before that opening bracket. I suspect this might break compilation.

Reply:
@ericleich Thanks so much for pointing out this issue. I will remove ; before the opening bracket. And have a nice day. ^_^

Reply:
@ericleich  Done, thanks a lot. 😊